tests/integration/golang/aim/experiment/get_experiment_activity_test.go

```diff
@@ -90,13 +91,11 @@ func (s *GetExperimentActivityTestSuite) Test_Error() {
 	for _, tt := range tests {
 		s.T().Run(tt.name, func(T *testing.T) {
 			var resp api.ErrorResponse
-			err := s.AIMClient.DoGetRequest(
-				fmt.Sprintf(
-					"/experiments/%s/runs?limit=4", tt.ID,
-				),
-				&resp,
-			)
-			assert.Nil(s.T(), err)
+			assert.Nil(s.T(), s.AIMClient.WithQuery(map[any]any{
+				"limit": 4,
```

Comment:
yesssss, more request this is in my plan but in scope of refactoring. I can do it even right now, but thennnn, you know, Dave won't be happy because of size of the PR =)

Reply:
yep, we can use `string` as a key.